project/config/ppsni/config/core.entity_form_display.node.page.default.yml

```diff
@@ -1,119 +1,121 @@
-uuid: 9330d8cb-7805-44d6-86f3-18b414fe5ae4
+uuid: 0b36753c-5b56-4b63-b3a1-50aa50ac337e
 langcode: en
 status: true
 dependencies:
   config:
-    - field.field.node.basic_page.body
-    - field.field.node.basic_page.field_meta_tags
-    - field.field.node.basic_page.field_next_audit_due
-    - node.type.basic_page
-    - workflows.workflow.nics_editorial_workflow
+    - field.field.node.page.body
+    - field.field.node.page.field_enable_toc
+    - field.field.node.page.field_meta_tags
+    - field.field.node.page.field_next_audit_due
+    - node.type.page
```

Comment:
Are you sure about this? All of the other Unity sites that I have looked at so far use the machine name 'basic_page', so changing it may have unexpected side effects